diff --git a/node_modules/@types/react-redux/index.d.ts b/node_modules/@types/react-redux/index.d.ts
index 6ef2715..7d89cd9 100644
--- a/node_modules/@types/react-redux/index.d.ts
+++ b/node_modules/@types/react-redux/index.d.ts
@@ -39,7 +39,7 @@ import {
     Store
 } from 'redux';
 
-import hoistNonReactStatics = require('hoist-non-react-statics');
+import { NonReactStatics } from '../hoist-non-react-statics';
 
 // Omit taken from https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-8.html
 export type Omit<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>;
@@ -96,7 +96,7 @@ export type GetProps<C> = C extends ComponentType<infer P> ? P : never;
 export type ConnectedComponentClass<
     C extends ComponentType<any>,
     P
-> = ComponentClass<JSX.LibraryManagedAttributes<C, P>> & hoistNonReactStatics.NonReactStatics<C> & {
+> = ComponentClass<JSX.LibraryManagedAttributes<C, P>> & NonReactStatics<C> & {
     WrappedComponent: C;
 };
 
